tensorflow/compiler/tf2tensorrt/convert/utils.h

```diff
@@ -73,6 +79,29 @@ struct VectorTensorShapeHasher {
    (NV_TENSORRT_MAJOR == major && NV_TENSORRT_MINOR == minor && \
     NV_TENSORRT_PATCH == patch && NV_TENSORRT_BUILD >= build))
 
+// This utility template converts an arithmetic type to a string.
+// This function is necessary to have the function:
+// `string DebugString(const std::vector<CType>&)` able to behave recursively.
+template<
+    typename CType,
+    typename = typename std::enable_if<
+      std::is_arithmetic<CType>::value, CType
+    >::type
+>
+string DebugString(const CType& el) {
+  string el_str = std::to_string(el);
+  el_str.erase(el_str.find_last_not_of('0') + 1, std::string::npos);
```

Comment:
Can you explain what does char '0' indicate? this line find the last not '0', and erase what goes after it.

Reply:
Can you add this explanation as a comment here?

Reply:
Sure :)